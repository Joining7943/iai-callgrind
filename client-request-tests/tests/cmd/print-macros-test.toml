bin.name = "valgrind-wrapper"
args = [
  "--tool=callgrind",
  "--valgrind-args=--verbose",
  "--bin=print-macros-test",
]
status = { code = 1 }
fs.sandbox = true
stdout = ""
