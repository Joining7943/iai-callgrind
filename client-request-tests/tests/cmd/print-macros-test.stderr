printf: foo

printf unchecked: foo
printf unchecked (invalid): INV
println: foo
println unchecked: foo
println unchecked (invalid): INV
printf backtrace: foo
at <__FILTER__>: VALGRIND_PRINTF_BACKTRACE.constprop.0 (valgrind.h:<__FILTER__>)
by <__FILTER__>: valgrind_printf_backtrace (native.c:<__FILTER__>)
by <__FILTER__>: print_macros_test::main (in <__FILTER__>)
by <__FILTER__>: std::sys_common::backtrace::__rust_begin_short_backtrace (in <__FILTER__>)
by <__FILTER__>: std::rt::lang_start::{{closure}} (in <__FILTER__>)
by <__FILTER__>: std::rt::lang_start_internal (function.rs:<__FILTER__>)
by <__FILTER__>: main (in <__FILTER__>)

at <__FILTER__>: VALGRIND_PRINTF_BACKTRACE.constprop.0 (valgrind.h:<__FILTER__>)
by <__FILTER__>: valgrind_printf_backtrace (native.c:<__FILTER__>)
by <__FILTER__>: print_macros_test::main (in <__FILTER__>)
by <__FILTER__>: std::sys_common::backtrace::__rust_begin_short_backtrace (in <__FILTER__>)
by <__FILTER__>: std::rt::lang_start::{{closure}} (in <__FILTER__>)
by <__FILTER__>: std::rt::lang_start_internal (function.rs:<__FILTER__>)
by <__FILTER__>: main (in <__FILTER__>)
printf backtrace unchecked: foo
at <__FILTER__>: VALGRIND_PRINTF_BACKTRACE.constprop.0 (valgrind.h:<__FILTER__>)
by <__FILTER__>: valgrind_printf_backtrace (native.c:<__FILTER__>)
by <__FILTER__>: print_macros_test::main (in <__FILTER__>)
by <__FILTER__>: std::sys_common::backtrace::__rust_begin_short_backtrace (in <__FILTER__>)
by <__FILTER__>: std::rt::lang_start::{{closure}} (in <__FILTER__>)
by <__FILTER__>: std::rt::lang_start_internal (function.rs:<__FILTER__>)
by <__FILTER__>: main (in <__FILTER__>)
printf backtrace unchecked (invalid): INV   at <__FILTER__>: VALGRIND_PRINTF_BACKTRACE.constprop.0 (valgrind.h:<__FILTER__>)
by <__FILTER__>: valgrind_printf_backtrace (native.c:<__FILTER__>)
by <__FILTER__>: print_macros_test::main (in <__FILTER__>)
by <__FILTER__>: std::sys_common::backtrace::__rust_begin_short_backtrace (in <__FILTER__>)
by <__FILTER__>: std::rt::lang_start::{{closure}} (in <__FILTER__>)
by <__FILTER__>: std::rt::lang_start_internal (function.rs:<__FILTER__>)
by <__FILTER__>: main (in <__FILTER__>)

at <__FILTER__>: VALGRIND_PRINTF_BACKTRACE.constprop.0 (valgrind.h:<__FILTER__>)
by <__FILTER__>: valgrind_printf_backtrace (native.c:<__FILTER__>)
by <__FILTER__>: print_macros_test::main (in <__FILTER__>)
by <__FILTER__>: std::sys_common::backtrace::__rust_begin_short_backtrace (in <__FILTER__>)
by <__FILTER__>: std::rt::lang_start::{{closure}} (in <__FILTER__>)
by <__FILTER__>: std::rt::lang_start_internal (function.rs:<__FILTER__>)
by <__FILTER__>: main (in <__FILTER__>)
println backtrace: foo
at <__FILTER__>: VALGRIND_PRINTF_BACKTRACE.constprop.0 (valgrind.h:<__FILTER__>)
by <__FILTER__>: valgrind_printf_backtrace (native.c:<__FILTER__>)
by <__FILTER__>: print_macros_test::main (in <__FILTER__>)
by <__FILTER__>: std::sys_common::backtrace::__rust_begin_short_backtrace (in <__FILTER__>)
by <__FILTER__>: std::rt::lang_start::{{closure}} (in <__FILTER__>)
by <__FILTER__>: std::rt::lang_start_internal (function.rs:<__FILTER__>)
by <__FILTER__>: main (in <__FILTER__>)
println backtrace unchecked: foo
at <__FILTER__>: VALGRIND_PRINTF_BACKTRACE.constprop.0 (valgrind.h:<__FILTER__>)
by <__FILTER__>: valgrind_printf_backtrace (native.c:<__FILTER__>)
by <__FILTER__>: print_macros_test::main (in <__FILTER__>)
by <__FILTER__>: std::sys_common::backtrace::__rust_begin_short_backtrace (in <__FILTER__>)
by <__FILTER__>: std::rt::lang_start::{{closure}} (in <__FILTER__>)
by <__FILTER__>: std::rt::lang_start_internal (function.rs:<__FILTER__>)
by <__FILTER__>: main (in <__FILTER__>)
println backtrace unchecked (invalid): INV   at <__FILTER__>: VALGRIND_PRINTF_BACKTRACE.constprop.0 (valgrind.h:<__FILTER__>)
by <__FILTER__>: valgrind_printf_backtrace (native.c:<__FILTER__>)
by <__FILTER__>: print_macros_test::main (in <__FILTER__>)
by <__FILTER__>: std::sys_common::backtrace::__rust_begin_short_backtrace (in <__FILTER__>)
by <__FILTER__>: std::rt::lang_start::{{closure}} (in <__FILTER__>)
by <__FILTER__>: std::rt::lang_start_internal (function.rs:<__FILTER__>)
by <__FILTER__>: main (in <__FILTER__>)

at <__FILTER__>: VALGRIND_PRINTF_BACKTRACE.constprop.0 (valgrind.h:<__FILTER__>)
by <__FILTER__>: valgrind_printf_backtrace (native.c:<__FILTER__>)
by <__FILTER__>: print_macros_test::main (in <__FILTER__>)
by <__FILTER__>: std::sys_common::backtrace::__rust_begin_short_backtrace (in <__FILTER__>)
by <__FILTER__>: std::rt::lang_start::{{closure}} (in <__FILTER__>)
by <__FILTER__>: std::rt::lang_start_internal (function.rs:<__FILTER__>)
by <__FILTER__>: main (in <__FILTER__>)

Start dumping at BB <__FILTER__> (Prg.Term.)...
Dump to
Dumping done.
Events    : Ir
Collected :

I   refs:
